src/libraries/System.Private.CoreLib/src/System/Resources/ResourceSet.cs

```diff
@@ -115,10 +116,11 @@ public virtual Type GetDefaultReader()
         // Returns the preferred IResourceWriter class for this kind of ResourceSet.
         // Subclasses of ResourceSet using their own Readers &; should override
         // GetDefaultReader and GetDefaultWriter.
+        // TODO: https://github.com/mono/linker/issues/943
+        [DynamicDependency(DynamicallyAccessedMemberTypes.PublicConstructors, "System.Resources.ResourceWriter", "System.Resources.Writer")]
         public virtual Type GetDefaultWriter()
         {
-            Assembly resourceWriterAssembly = Assembly.Load("System.Resources.Writer, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
-            return resourceWriterAssembly.GetType("System.Resources.ResourceWriter", throwOnError: true)!;
+            return Type.GetType("System.Resources.ResourceWriter, System.Resources.Writer", throwOnError: true)!;
```

Comment:
@jkotas split this into the 2 calls in [https://github.com/dotnet/coreclr/pull/11314/files#diff-37ee6a4a589ffae33ce341f9c1a28133R136](https://github.com/dotnet/coreclr/pull/11314/files#diff-37ee6a4a589ffae33ce341f9c1a28133R136). Are we sure this is safe to collapse it back into a single `Type.GetType` call?

Reply:
The PR was reconciling differences between multiple forked copies of the same code. I do not think there was a strong reason why I have changed this to `Assembly.Load` + `GetType` that was used in the other fork.

It should be ok to change it back to `Type.GetType`.